id_infrastructure/firestore_uuid_table.py

```diff
@@ -223,9 +223,17 @@ def uuid_to_data_batch(self, uuids_to_lookup):
         # Return a mapping data for the uuids that were in the collection
         log.info(f"Looking up the data for {len(uuids_to_lookup)} uuids from Firestore...")
         reverse_mappings = dict()
-        for mapping in self._mappings_ref().get():
-            self._mappings_cache[mapping.id] = mapping.get(_UUID_KEY_NAME)
-            reverse_mappings[mapping.get(_UUID_KEY_NAME)] = mapping.id
+
+        mappings_ref = self._mappings_ref()
+
+        # Fetch the entire mappings table from Firestore, in batches of 500 mappings at a time so large tables don't
+        # time out.
+        batch = mappings_ref.limit(500).get()
```

Comment:
Define limit 500 as a constant to enhance readability.

Reply:
Done - this was actually already a constant at the top of the file, so using that now.